lib/checkWipDraftPR.js

```diff
@@ -41,26 +61,28 @@ module.exports.checkWIP = async (context) => {
   const prAuthor = pullRequest.user.login;
 
   if (isDraftPr(pullRequest) || isWIPPr(pullRequest)) {
-    const link = 'here'.link(
-      'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia' +
-      '#wip--draft-pull-requests');
-    // Comment on Pull Request.
-    const commentBody = (
-      'Hi @' + prAuthor + ', WIP/Draft PRs are highly discouraged. You can ' +
-      'learn more about it ' + link + '. You can reopen it when it\'s ' +
-      'ready to be reviewed and ensure that it is without any WIP phrase ' +
-      'in title or body. Thanks!');
-    const commentParams = context.repo({
-      issue_number: pullRequest.number,
-      body: commentBody,
-    });
-    await context.github.issues.createComment(commentParams);
+    const hasSkipCIMessage = await isSkipCICommit(context);
+    if (!hasSkipCIMessage) {
+      const link = 'here'.link(
+        'https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia' +
+        '#wip--draft-pull-requests');
```

Comment:
Have you updated this documentation as well?

Reply:
Yes I did https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#wip--draft-pull-requests

Reply:
Add a link to the travis and circle ci documentation as well in the wiki:

1. https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build
2.  https://circleci.com/docs/2.0/skip-build/#skipping-a-build